configs/common/packages.yaml

```diff
@@ -294,4 +305,5 @@ packages:
   wrf-io:
     require: '@1.2.0'
   zstd:
-    require: '@1.5.2 +programs'
+    #require: '@1.5.2 +programs'
```

Comment:
Is there a reason not to pin the zstd version? It's not a huge issue, but I would tend to lean toward pinning versions for packages that are used by UWM, in other words I would generally put zstd in the same category as hdf5, netcdf, etc.

Reply:
Noted. I did set `zstd` to the version that got built in my and your tests (1.5.6) in my latest commit.